### PR TITLE
remove SIMULATOR variable from icarus/ and verilator/ Makefile

### PR DESCRIPTION
**Pull request by codingUniv**
It is already defined in config.mk

### CODE REVIEW

**Comment by jjts** [comment]
This PR is content less

I suggest you work on the README which is a REAL problem

**Comment by codingUniv** [comment]
I tried in both icarus and verilator directories, simulation works without setting SIMULATOR variable in their own Makefile files.

**Comment by codingUniv** [comment]
First README updates will be submitted soon

File: hardware/simulation/icarus/Makefile [hunk 1]
```diff
@@ -4,13 +4,11 @@ CACHE_DIR:=../../..
...
 defmacro:=-D
 
-SIMULATOR=icarus
-
 include ../simulation.mk
 
 #testbench
 TB ?= $(CACHE_TB_DIR)/pipeline-iob-cache_tb.v
-#TB ?=$(CACHE_TB_DIR)/rep_pol_tb.v
+#TB ?=$(CACHE_TB_DIR)/rep_pol_tb.v"
```
Comment (jjts) [inline hardware/simulation/icarus/Makefile:11]: Now, instead of an 'e' you have a '"'

Comment (codingUniv) [reply hardware/simulation/icarus/Makefile]: It is now fixed

**Pull request merged**